webapp/components/admin_console/reset_password_modal.jsx

```diff
@@ -3,17 +3,10 @@
 
 import ReactDOM from 'react-dom';
 import Client from 'client/web_client.jsx';
-import Constants from 'utils/constants.jsx';
+import * as Utils from 'utils/utils.jsx';
 import {Modal} from 'react-bootstrap';
 
-import {injectIntl, intlShape, defineMessages, FormattedMessage} from 'react-intl';
-
-var holders = defineMessages({
-    submit: {
-        id: 'admin.reset_password.submit',
-        defaultMessage: 'Please enter at least {chars} characters.'
-    }
-});
+import {injectIntl, intlShape, FormattedMessage} from 'react-intl';
```

Comment:
it seems to me that now you can remove the `injectIntl` and `intlShape`  variables from the import